code/modules/mob/living/carbon/xenomorph/castes/runner/castedatum_runner.dm

```diff
@@ -151,3 +150,46 @@
 
 	// *** Ranged Attack *** //
 	pounce_delay = 13 SECONDS
+
+/datum/xeno_caste/runner/primordial
+	upgrade_name = "Primordial"
+	caste_desc = "Not what you want to run into in a dark alley. It looks extremely deadly."
+	primordial_message = "We are the fastest assassin of all time. Our speed is unmatched."
```

Comment:
Runner
Desc: "A sprinting terror of the hive. It looks ancient and menacing."
Message: "Nothing can outrun us. We are the swift death."